book/website/reproducible-research/rdm/rdm-elns.md

```diff
@@ -0,0 +1,234 @@
+Mendeley for example began [encrypting their local database](https://www.zotero.org/support/kb/mendeley_import#mendeley_database_encryption) making it impossible to migrate your local library to different reference managers without going through their online library feature.
+This occured after they had reached a high degree of market penetration and been aquired by Elsevier.
+
+When looking for any critical piece of software for long term use the first questions to ask are:
+- Is there an Libre / open-source solution to this problem?
+- If it is a web app and/or has a server: Could I host my own fully featured instance, should I need to?
+- Is there a large/active community using the project, does it have institutional backing of some kind? 
+  This might take the form of a company which sells service contracts, or offers paid hosting ideally with feature parity with a self-hosted option. 
+  Or perhaps a foundation or other non-profit/academic organisation with robust funding.
+- What are the data export options and formats?
+
+Open solutions provide you with the assurance that if you do the appropriate preparatory work you should be able to access all of your data in it's native form.
+For example keeping copies of the ELN software that can be run offline in a VM or similar computational environment in the future.
+This is typically much easier for open applications and can be impossible for proprietary ones.
+Even if the tools are no longer maintained and in a state that can be used in production they can still be used to read the data and interact with it in (mostly) the same way.
+The data will also likely be stored in an open format from which it can relatively easily be extracted and ported to a new format.
+This may still be a lot of work but it is much more attainable than reverse engineering a proprietary format for which there is no documentation or worse - dealing with encrypted files.
+
+You can pay for third party hosting and administration of open source applications.
+This is a solution when you don't have the expertise or internal resources to administer a self-hosted instance of an open-source ELN solution. 
+In this case you get the best of both worlds, the benefits of professional support and the reassurance of an open solution.
+You should still take regular local backups of exports from your hosting provider from which you could restore your ELN system with different hosting.
+This means that you retain the option to change providers.
+Conventionl software as a service (SaaS) platforms are highly verically integrated, the software development, hosting, and administration are package deal that you must take or leave as is.
+Using open tools breaks this up affording you the flexibility to change any of these if the need arises and you have the resources to do so, it puts you in a better bargining position.
```

Comment:
```suggestion
Using open tools breaks this up, affording you the flexibility to change any of these if the need arises.
```
Or a different edit to make this sentence a bit more readable

Reply:
Addressed in: [bfbcaec](https://github.com/the-turing-way/the-turing-way/pull/3466/commits/bfbcaec15b27994c94556acb918eaa143a4dcf28)